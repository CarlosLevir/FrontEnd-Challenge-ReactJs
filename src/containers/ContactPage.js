import React from 'react';

const ContactPage = () => (
  <div>
    <h3>Here will be the contact page</h3>
  </div>
);

export default ContactPage;
