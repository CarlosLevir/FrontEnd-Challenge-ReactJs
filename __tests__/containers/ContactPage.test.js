import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import ContactPage from '../../src/containers/ContactPage';

configure({ adapter: new Adapter() });

describe('ContactPage', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ContactPage />).shallow();
    expect(wrapper).toMatchSnapshot();
  });
});
