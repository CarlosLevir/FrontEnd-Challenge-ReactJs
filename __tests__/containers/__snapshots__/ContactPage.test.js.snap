// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <div>
    <h3>
      Here will be the contact page
    </h3>
  </div>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3>
        Here will be the contact page
      </h3>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Here will be the contact page",
      },
      "ref": null,
      "rendered": "Here will be the contact page",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3>
          Here will be the contact page
        </h3>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Here will be the contact page",
        },
        "ref": null,
        "rendered": "Here will be the contact page",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
